lib/livebook/storage/ets.ex

```diff
@@ -0,0 +1,110 @@
+defmodule Livebook.Storage.Ets do
+  @moduledoc """
+  Ets implementation of `Livebook.Storage` behaviour.
+
+  The module is supposed to be started just once as it
+  is responsible for managing a single global named ets table.
+
+  `insert` and `delete` operations are supposed to be called using the genserver
+  while all the lookups can be performed by directly accessing the named table.
+  """
+  @behaviour Livebook.Storage
+
+  @table_name :__livebook_storage__
```

Comment:
```suggestion
  @table_name __MODULE__
```

:) This is nice because the name of the table is the name of the module, it may help when debugging/introspecting.